### PR TITLE
[14.0][ADD] project_forecast_line_milestone

### PR DESCRIPTION
**Pull request by ntsirintanis**

### CODE REVIEW

File: project_forecast_line_milestone/models/project_task.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    def _compute_forecast_date_planned_end(self):
+        """Override method to use milestone_id.target_date"""
+        for this in self:
+            if this.milestone_id.target_date:
+                this.forecast_date_planned_end = this.milestone_id.target_date
+                continue
+            if this.date_deadline:
+                this.forecast_date_planned_end = this.date_deadline
+                continue
+            this.forecast_date_planned_end = this.forecast_date_planned_end
```
Comment (ntsirintanis) [inline project_forecast_line_milestone/models/project_task.py]: This appears to be possible in 14.0

Comment (gjotten) [reply project_forecast_line_milestone/models/project_task.py]: @ntsirintanis yup seems to go quite well!

**Review by gjotten — APPROVED** [review]

**Review by thomaspaulb — APPROVED** [review]

File: project_forecast_line_milestone/models/project_task.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        return super()._forecast_date_planned_end_depends_list() + [
+            "milestone_id",
+            "milestone_id.target_date",
+        ]
+
+    @api.depends(_forecast_date_planned_end_depends_list)
+    def _compute_forecast_date_planned_end(self):
+        """Override method to use milestone_id.target_date"""
+        res = super()._compute_forecast_date_planned_end()
+        for this in self:
```
Comment (ntsirintanis) [inline project_forecast_line_milestone/models/project_task.py]: change this to task in self

**Comment by thomaspaulb** [comment]
@ntsirintanis This non-OCA MR can probably be closed, no?

**Comment by ntsirintanis** [comment]
> @ntsirintanis This non-OCA MR can probably be closed, no?

yes, it is merged now

**Pull request closed without merging**